client/lib/directly/index.js

```diff
@@ -14,45 +20,64 @@ const DEFAULT_RTM_WIDGET_OPTIONS = {
 };
 const DIRECTLY_RTM_SCRIPT_URL = 'http://widgets.wp.com/directly/embed.js';
 const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
-let initializationPromise;
+let _initialize;
+
+/**
+ * Set up global variables and configuration for the Directly library.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S
+ */
+function configureDirectlyGlobals() {
+	// Set up the global DirectlyRTM function, required for the Directly library.
+	// This snippet is pasted from Directly's setup code.
```

Comment:
:+1: As I understand Directly the company offers a single product, the Real-Time Messaging (RTM) widget. I'll do a pass through the lib and docs to make it less confusing.